includes/reader-activation/class-reader-activation-emails.php

```diff
@@ -76,7 +76,7 @@ public static function add_email_configs( $configs ) {
 		$configs[ self::EMAIL_TYPES['MAGIC_LINK'] ]     = [
 			'name'                   => self::EMAIL_TYPES['MAGIC_LINK'],
 			'label'                  => __( 'Login link', 'newspack' ),
-			'description'            => __( 'Email with a login link.', 'newspack' ),
+			'description'            => __( 'Email sent to users with a login link generated by Admin user or when OTP is disabled.', 'newspack' ),
```

Comment:
Nit:
```suggestion
			'description'            => __( 'Email sent to users with a login link generated by Admin user or when one-time password is disabled.', 'newspack' ),
```
We're using 'one-time password' instead of OTP in the other two spots it appears in this section -- what do you think about switching this one, too, for consistency?

Reply:
Good call! Updated in https://github.com/Automattic/newspack-plugin/pull/3323/commits/897b26201435982fb5024248f7f41cf0e7c81761